string-constants-doc/string-constants/translation-guide.scrbl

```diff
@@ -0,0 +1,161 @@
+#lang scribble/manual
+
+@title{How To Translate DrRacket's String Constants in Your Language}
+@author[(author+email "Alexander Shopov" "ash@kambanaria.org"  #:obfuscate? #t)]
+
+This is a short hands on guide how to translate the strings used
+within DrRacket's GUI. It is targeted at people willing to translate
+the interface who may not know all ins and outs of Racket or Scheme.
+
+@; ----------------------------------------------------------------------
+
+@section{Introduction}
+
+DrRacket is a GUI: editor and runner for the many languages you can
```

Comment:
Usually, I would say "DrRacket is a development environment for ..."

Reply:
Done